### PR TITLE
docs(blog): Update hacktoberfest 2023 closing blog post

### PR DESCRIPTION
**Pull request by ShihChun-H**
Because

- Hacktoberfest has come to the end

This commit

- update the Hacktoberfest 2023 closing blog post
- update the contributors' name

### CODE REVIEW

**Review by xiaofei-du — APPROVED** [review]

File: blog/hacktoberfest-2023-closing.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- **Rutam Mishra** - https://twitter.com/RutamHere/status/1713732255821242867
+    - Text Sentiment Analysis
+- **Rutam Mishra** - https://twitter.com/RutamHere/status/1713738373041652057
+    - Accommodation Recommendation given the location and star desired
+- **Rutam Mishra** - https://twitter.com/RutamHere/status/1713761397753209078
+    - Chemical Element Inquiry with the element as the input and the symbol, atomic mass and atomic number as the output
+- **Rutam Mishra** - https://twitter.com/RutamHere/status/1713887508172509268
+    - SPAM / HAM identification
+- **Rutam Mishra** - https://twitter.com/RutamHere/status/1714392688437735607
+    - Workout Procedure Generator
```
Comment (xiaofei-du) [inline blog/hacktoberfest-2023-closing.mdx:58]: Same as above. If these wonderful contributors have their GitHub accounts, we can add them.

File: blog/hacktoberfest-2023-closing.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Social Media Sharing
+
+- **Andre Gabriel**
+- **Soumyadip Moni**
+- **Ayroti Dey Sarkar**
+- **Ján Timoranský**
+- **Aviral Kaintura**
+- **Sarthak Tyagi**
+- **Syed Zubeen**
```
Comment (xiaofei-du) [inline blog/hacktoberfest-2023-closing.mdx:69]: Same as above

File: blog/hacktoberfest-2023-closing.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+During this event, we witnessed remarkable achievements and milestones reached by our participants. 
+
+The top 30% contributors will be receiving an exclusive **Instill AI swag package**, including physical stickers and a choice of an Instill bottle or mug.
+
+- **Swag Packages Winners**:
+    - **Rutam Mishra**
+    - **Syed Zubeen**
+    - **Vibhor Bhatt**
+    - **Miguel Ortiz**
+    - **Sajda Kabir**
```
Comment (xiaofei-du) [inline blog/hacktoberfest-2023-closing.mdx:85]: Same as above

**Review by xiaofei-du — CHANGES_REQUESTED** [review]
Please see the above comments.

**Comment by xiaofei-du** [comment]
Please remove this half-down article. Curious why you did not directly edit this article, but instead made a new one.

image

File: blog/hacktoberfest-2023-closing.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## List of Contributors
+
+Here is a list of contributors who made outstanding contributions during the Instill AI x Hacktoberfest 2023:
+
+### Code Contributing
+
+- **Vibhor Bhatt** - https://github.com/instill-ai/model-backend/pull/436
+- **Miguel Ortiz** - https://github.com/instill-ai/operator/pull/16
+- **Sajda Kabir** - https://github.com/instill-ai/instill.tech/pull/625
```
Comment (xiaofei-du) [inline blog/hacktoberfest-2023-closing.mdx:37]: Please provide a brief summary of each PR contribution to the readers.

File: blog/hacktoberfest-2023-closing.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+
+### Tutorial Creating
+
+- **Rutam Mishra** - https://dev.to/rutamhere/building-an-instill-ai-pipeline-in-5-minutes-4fp2
```
Comment (xiaofei-du) [inline blog/hacktoberfest-2023-closing.mdx:42]: Please provide a brief summary of this tutorial to the readers.

**Review by xiaofei-du — CHANGES_REQUESTED** [review]
@ShihChun-H, I have added extra comments, please check.

File: blog/hacktoberfest-2023-closing.mdx (hunk 4 above)
Comment (ShihChun-H) [reply blog/hacktoberfest-2023-closing.mdx:37]: Hi @xiaofei-du, I did not create a new one. I changed the name of the file/title, maybe that is why it is identified as a new one. From this branch, I only see one closing blog post, basically, it is the same one.

File: blog/hacktoberfest-2023-closing.mdx [hunk 6]
```diff
@@ -0,0 +1,104 @@
+---
+title: "Hacktoberfest 2023: Thank You and See You Next Year"
+lang: "en-US"
+draft: false
+description: "As the month-long coding extravaganza comes to a close, we want to express our heartfelt gratitude to all participants of the Instill AI x Hacktoberfest 2023! It has been an incredible journey of collaboration, learning, and community spirit. Let's take a moment to reflect on the amazing accomplishments and recognize the outstanding contributions made during this event."
+slug: "hacktoberfest2023-closing"
+publishedOn: "2023-10-23T18:00:00"
```
[the remark this answers is not in the document]
Comment (ShihChun-H) [reply blog/hacktoberfest-2023-closing.mdx:7]: Hi @xiaofei-du, can you further explicit which part needs to be updated? Thanks. Other than `PublishedOn`, it looks good to me.

**Comment by ShihChun-H** [comment]
> Please remove this half-down article. Curious why you did not directly edit this article, but instead made a new one.
> 
> image

Hi @xiaofei-du , I did not create a new one. I changed the name of the file/title, maybe that is why it is identified as a new one. From this branch, I only see one closing blog post, basically, it is the same one.

File: blog/hacktoberfest-2023-closing.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (xiaofei-du) [reply blog/hacktoberfest-2023-closing.mdx]: yeah, the publish date

**Review by xiaofei-du — APPROVED** [review]

**Pull request merged**